src/core/client/admin/components/UserRole/SiteModeratorModal.tsx

```diff
@@ -41,15 +41,21 @@ const SiteModeratorModal: FunctionComponent<Props> = ({
   query,
 }) => {
   const onSubmit = useCallback(
-    async (values: { siteIDs: string[] }) => {
+    async (values: { siteIDs: string[] }, actions) => {
       try {
         await onFinish(values.siteIDs);
         return;
       } catch (err) {
         if (err instanceof InvalidRequestError) {
-          return err.invalidArgs;
+          actions.setStatus({
```

Comment:
formik doesn't have a `submitError` field or something comparable, so we use the `status` field to store server errors that aren't associated with a single field.